src/App/use-check-updates.ts

```diff
@@ -0,0 +1,24 @@
+import { message } from 'antd';
+import { useEffect } from 'react';
+
+async function checkForUpdates() {
+  try {
+    const newerVersion = await window.checkForUpdates();
+    if (newerVersion) {
+      message.info({
+        content: `New version ${newerVersion} available! See Help -> Latest Releases`,
```

Comment:
The "programmer's arrow" is not something I'd present to a regular user.

Reply:
suggestions? drop altogether? nothing comes to mind :/

Reply:
If we put a link to releases then we can remove this altogether.

Reply:
`See Help ► Latest Releases` maybe?

Reply:
supposedly `>` is a standard, I found a thread citing https://support.apple.com/en-us/HT201251#authorize

Reply:
Works for me. There are also one-character arrows, or a slash etc.